packages/multichain/src/scope/types.ts

```diff
@@ -0,0 +1,103 @@
+import MetaMaskOpenRPCDocument from '@metamask/api-specs';
+import {
+  isCaipNamespace,
+  isCaipChainId,
+  parseCaipChainId,
+} from '@metamask/utils';
+import type {
+  CaipChainId,
+  CaipReference,
+  CaipAccountId,
+  KnownCaipNamespace,
+  CaipNamespace,
+  Json,
+} from '@metamask/utils';
+
+export enum KnownWalletScopeString {
+  Eip155 = 'wallet:eip155',
+}
+
+export type NonWalletKnownCaipNamespace = Extract<
+  KnownCaipNamespace,
+  KnownCaipNamespace.Eip155
+>;
+
+export const KnownWalletRpcMethods: string[] = [
+  'wallet_registerOnboarding',
+  'wallet_scanQRCode',
+];
+const WalletEip155Methods = ['wallet_addEthereumChain'];
+
+const Eip155Methods = MetaMaskOpenRPCDocument.methods
```

Comment:
The type on this would be `string[]`. Similar comment as above, do we want to keep the type in case it's useful?

Reply:
makes the types too narrow and difficult to consume downstream. Same problem as https://github.com/MetaMask/core/pull/4784#discussion_r1813116599